inspirehep/modules/records/receivers.py

```diff
@@ -388,3 +388,14 @@ def generate_name_variations(sender, json, *args, **kwargs):
                 'output': full_name,
                 'payload': {'bai': bais[0] if bais else None}
             }})
+
+
+def populate_author_count(sender, json, *args, **kwargs):
+    """Populate author-count using only authors with 'inspi
+    re_roles' not equal to 'supervisor'."""
```

Comment:
Relevant portion of PEP 0257 (https://www.python.org/dev/peps/pep-0257/#one-line-docstrings):
> One-liners are for really obvious cases. They should really fit on one line.